apps/dotcms-ui/src/app/view/components/dot-content-compare/components/dot-content-compare-table/dot-content-compare-table.component.ts

```diff
@@ -21,5 +22,40 @@ export class DotContentCompareTableComponent {
         { label: this.dotMessageService.get('plain'), value: false }
     ];
 
-    constructor(private dotMessageService: DotMessageService) {}
+    constructor(
+        private dotMessageService: DotMessageService,
+        private dotFormatDateService: DotFormatDateService
+    ) {}
+
+    /**
+     * Formats the label from the modDate, to show the relative date if modDate less than a week
+     * and author
+     *
+     * @param {DotCMSContentlet} item
+     * @returns {string}
+     * @memberof DotContentCompareTableComponent
+     */
+    transformVersionItemLabel(item: DotCMSContentlet): string {
```

Comment:
This method is doing too much.

I propose one to transform the date and one to get the label.